.circleci/config.yml

```diff
@@ -19,40 +19,52 @@ jobs:
       - run:
           name: "Install required packages"
           command: |
-            sudo add-apt-repository -y ppa:projectatomic/ppa
+            source /etc/os-release
+            sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+            wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${VERSION_ID}/Release.key -O- | sudo apt-key add -
             sudo apt-get update
-            sudo apt-get install libkrb5-dev software-properties-common uidmap podman docker-ce libxml2-dev libxslt1-dev
+            sudo apt-get install libkrb5-dev software-properties-common containerd runc podman docker.io
```

Comment:
Switch from `docker-ce` to `docker.io` and add `podman`. Also removed unnecessary libraries.